xcache/image-config.d/00-functions.sh

```diff
@@ -0,0 +1,25 @@
+supervisord_is_enabled () {
+    local service="$1"
+    [[ -e /etc/supervisord.d/${service}.conf ]]
+}
+
+
+supervisord_is_disabled () {
+    local service="$1"
+    [[ -e /etc/supervisord.d/${service}.conf.disabled ]]
+}
+
+
+supervisord_enable () {
+    local service="$1"
+    if supervisord_is_disabled ${service}; then
+        mv -f /etc/supervisord.d/${service}.conf.disabled /etc/supervisord.d/${service}.conf
```

Comment:
Totally fine how you have it; but if it makes it any easier to see what you're changing, do you like this any better or worse?

```suggestion
        mv -f /etc/supervisord.d/${service}.conf{.disabled,}
```

Reply:
If it were a specific service instead of `${service}` then I'd say no, because it would hurt greppability, but in this case I think it's fine.